packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -382,6 +385,8 @@ ComplexItem.propTypes = {
 	t: PropTypes.func,
 };
 
+export const ComplexItem = translate(I18N_DOMAIN_COMPONENTS)(ComplexItemUntranslated);
```

Comment:
Not found of ComplexItemUntranslated name.
The complex item is not exported and it's display name is complex item.
Every component will have the suffix untranslated with this logic.
Can we inverse the naming calling something translated than untranslated ?